### ISSUE TITLE
Cannot sign message from extensionProvider

### ISSUE THREAD

**Issue by zikunfan** [ElrondNetwork/elrond-sdk-erdjs#78]
## Description

We're trying to call `provider.signMessage()` with the following code:

```ts
const sm = new SignableMessage({
  message: Buffer.from('0x' + Buffer.from(data).toString('hex'), 'ascii'),
})
```
And send message with 
```ts
provider.signMessage(sm)
            .then(e => { return e.toJSON().toString()})
            .catch(err => { console.error(err); return ''; })
```

Then I got blank page then I received an error

```
Extension window was closed without response.
```

## Investigation

We read the source code, found `signMessage` called `startExtMsgChannel`. Then we debug this function found error may happends https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/main/src/dapp/extensionProvider.ts#L204-L208
Then we cannot go deeper, cause the extension wallet seems not open sourced.

**Comment by camilbancioiu** [comment]
Hello! We have a fix by @andreigiura in the pipeline, it will be part of the upcoming release.

**Comment by andreigiura** [comment]
This issue was fixed in erdjs v8.0.1-alpha6, Maiar DeFi Wallet 1.0.6. Closing issue.
